salt/cert/init.sls

```diff
@@ -7,3 +7,22 @@ include:
          pillar['ssl']['crt_file'],
          pillar['ssl']['key_file'],
          o = pillar['certificate_information']['subject_properties']['O']) }}
+
+#######################################
+# additional system wide certificates #
+#######################################
+
+# Install additional certificates that were setup in Velum by the user as
+# system-wide certificates
+
+{% set certs = salt.caasp_pillar.get('system_certificates', []) %}
+{% for cert_tuple in certs %}
+  {% set name, cert = cert_tuple %}
+
+/etc/pki/trust/anchors/{{ name }}.crt:
+  file.managed:
+    - makedirs: True
+    - contents: |
```

Comment:
We should explicitly set the user/group to root/root and mode to 644 or similar.